challenge_14/action_controller.rb

```diff
@@ -23,6 +24,44 @@ class Blog < ActiveRecord::Base; end
   end
 end
 
+class AppController < ActionController::Base
+  def root
+    response = ""
```

Comment:
The only thing to be slightly cautious of here is that `#response` is [an instance method](https://api.rubyonrails.org/classes/ActionController/Base.html#method-i-response) on `ActionController::Base`, so using the same name for a local variable is preventing you from calling it (with implicit receiver) if you want to. Not an actual problem, but another choice like `response_body` might be safer.